### ISSUE TITLE
Error: ActiveRecord::StatementInvalid: Could not find table 'action_text_rich_texts'

### ISSUE THREAD

**Issue by hrdwdmrbl** [gregnavis/active_record_doctor#43]
```
$ bundle exec rake active_record_doctor:missing_presence_validation
```
```
rake aborted!
ActiveRecord::StatementInvalid: Could not find table 'action_text_rich_texts'
~/.rvm/gems/ruby-2.6.6/gems/activerecord-6.0.3.2/lib/active_record/connection_adapters/sqlite3_adapter.rb:358:in `table_structure'
~/.rvm/gems/ruby-2.6.6/gems/activerecord-6.0.3.2/lib/active_record/connection_adapters/abstract/schema_statements.rb:114:in `columns'
~/.rvm/gems/ruby-2.6.6/gems/active_record_doctor-1.7.1/lib/active_record_doctor/tasks/missing_presence_validation.rb:14:in `block in run'
~/.rvm/gems/ruby-2.6.6/gems/active_record_doctor-1.7.1/lib/active_record_doctor/tasks/missing_presence_validation.rb:11:in `map'
~/.rvm/gems/ruby-2.6.6/gems/active_record_doctor-1.7.1/lib/active_record_doctor/tasks/missing_presence_validation.rb:11:in `run'
~/.rvm/gems/ruby-2.6.6/gems/active_record_doctor-1.7.1/lib/active_record_doctor/tasks/base.rb:7:in `run'
~/.rvm/gems/ruby-2.6.6/gems/active_record_doctor-1.7.1/lib/tasks/active_record_doctor.rake:17:in `block in mount'
~/.rvm/gems/ruby-2.6.6/gems/airbrake-10.0.2/lib/airbrake/rake.rb:19:in `execute'
~/.rvm/gems/ruby-2.6.6/gems/rake-13.0.1/exe/rake:27:in `<top (required)>'
~/.rvm/gems/ruby-2.6.6/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:63:in `load'
~/.rvm/gems/ruby-2.6.6/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:63:in `kernel_load'
~/.rvm/gems/ruby-2.6.6/gems/bundler-2.1.4/lib/bundler/cli/exec.rb:28:in `run'
~/.rvm/gems/ruby-2.6.6/gems/bundler-2.1.4/lib/bundler/cli.rb:476:in `exec'
~/.rvm/gems/ruby-2.6.6/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/command.rb:27:in `run'
~/.rvm/gems/ruby-2.6.6/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/invocation.rb:127:in `invoke_command'
~/.rvm/gems/ruby-2.6.6/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor.rb:399:in `dispatch'
~/.rvm/gems/ruby-2.6.6/gems/bundler-2.1.4/lib/bundler/cli.rb:30:in `dispatch'
~/.rvm/gems/ruby-2.6.6/gems/bundler-2.1.4/lib/bundler/vendor/thor/lib/thor/base.rb:476:in `start'
~/.rvm/gems/ruby-2.6.6/gems/bundler-2.1.4/lib/bundler/cli.rb:24:in `start'
~/.rvm/gems/ruby-2.6.6/gems/bundler-2.1.4/exe/bundle:46:in `block in <top (required)>'
~/.rvm/gems/ruby-2.6.6/gems/bundler-2.1.4/lib/bundler/friendly_errors.rb:123:in `with_friendly_errors'
~/.rvm/gems/ruby-2.6.6/gems/bundler-2.1.4/exe/bundle:34:in `<top (required)>'
~/.rvm/gems/ruby-2.6.6/bin/bundle:23:in `load'
~/.rvm/gems/ruby-2.6.6/bin/bundle:23:in `<main>'
Tasks: TOP => active_record_doctor:missing_presence_validation
(See full trace by running task with --trace)
```

**Comment by MatheusRich** [comment]
Same with `active_record_doctor:missing_non_null_constraint`

**Comment by olivierbuffon** [comment]
Any update on this issue by any chance? I'm experiencing the same problem as well.

**Comment by cxspxr** [comment]
same

**Comment by gregnavis** [comment]
Thank you for reporting. This is fixed in 14aba5697dfcc5e7647817376bfe92dbff0c40d8. The fix should be available later this week after I release 1.7.2.

**Comment by gregnavis** [comment]
I've just released 1.7.2 with the aforementioned fix. I'm closing the issue but if there are still issues feel free to reopen.
